### ISSUE TITLE
High memory usage on Windows

### ISSUE THREAD

**Issue by viktorix** [nextcloud/talk-desktop#886]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments.

---

## Steps to reproduce
1. Install Windows client.
2. Open it and leave it.
3. Check task manager for memory usage.

### Expected behaviour
Memory usage shouldn't be this high when compared to Brave that has 10 tabs open and Thunderbird. Ideally, when no calls happening, it should be kept closer to 500mb. It's not an exact number, just a recommendation based on other Electron apps I've seen.

![Image](https://github.com/user-attachments/assets/19503a84-5b84-4f11-a623-c692d42cd5a6)

### Actual behaviour
Having client open, without any calls, memory usage is over 1000MB.

## Desktop client

**Talk Desktop client version:** 1.0.0-rc.1

**Operating system:** Windows

**Operating system version:** Windows_NT 10.0.22631 (Windows 11 Home)

**Microphone available:** yes

**Camera available:** yes

## Server

This client connects to c.nc.c server.

## Notes

I know it's difficult to manage memory usage in Electron apps. So one consideration can be to introduce hibernation mode and setting, like many Electron apps do. User sets a specific time of inactivity, let's say 15 min, and if Talk isn't used in 15 min it hibernates to reduce memory usage. User needs to click a button to reload it (ex: [Rambox](https://rambox.app/features/hibernation/)).

**Comment by ShGKme** [comment]
> I know it's difficult to manage memory usage in Electron apps. So one consideration can be to introduce hibernation mode and setting, like many Electron apps do. User sets a specific time of inactivity, let's say 15 min, and if Talk isn't used in 15 min it hibernates to reduce memory usage. User needs to click a button to reload it (ex: [Rambox](https://rambox.app/features/hibernation/)).

I don't think this is Electron itself. RAM usage increases on opening huge chats and decreases after some time. Usually it is about 250-450 MB depending on the system.

**Comment by ShGKme** [comment]
Can confirm the issue
